MachObfuscator/SymbolsListParsing/SimpleSymbolsListLoader.swift

```diff
@@ -0,0 +1,40 @@
+import Foundation
+
+class SimpleSymbolsListLoader: ObjectSymbolsLoader {
```

Comment:
I did this having in mind existing `SimpleImageLoader` class... which was named that way because I had idea of something like `CachingImageLoader`. But you are right. Both those names are unfortunate. I like `TextFileSymbolListLoader`, will change the class name that way. Thanks!